entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -732,13 +690,21 @@ class EntityService(
                 expandJsonLdKey(propertyFragment.first, entity.contexts)!!,
                 propertyFragment.second, entity.contexts
             )
-            val entityEvent = EntityEvent(
-                operationType = EventType.UPDATE,
-                entityId = entity.id,
-                entityType = entity.type[0].extractShortTypeFromExpanded(),
-                payload = propertyPayload
-            )
-            applicationEventPublisher.publishEvent(entityEvent)
+
+            val jsonLdEntity = getFullEntityById(entity.id, true)
+            jsonLdEntity?.let {
+                entitiesEventService.publishEntityEvent(
```

Comment:
this has to be moved into web handler

Reply:
the function is used in the ObservationListener, should I move it there ?